python/modem-firmware-1.3+/README.md

```diff
@@ -47,6 +47,9 @@ If the `rtt` option is specified, communication will be performed using [SEGGER'
 To use RTT, the device must be running the [Modem Shell](https://github.com/nrfconnect/sdk-nrf/tree/main/samples/cellular/modem_shell) sample application [built with the RTT overlay](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/cellular/modem_shell/README.html#segger-rtt-support).
 This script will optionally flash the modem shell application on startup if a hex file path is provided with the `mosh_rtt_hex` option.
 
+**Note**: the device ID options must match those used to build the device firmware, or else the device will fail to connect to nRF Cloud.
```

Comment:
this is not true for user created certs.  the cloud doesn't actually use the CN.
it is "nice" to have it match, for reference.

it only applies to JITP certs (which are only created by the cloud).

Reply:
ah yes, i still head the top-most change in my head while i was reading this change 😅